include/odp/api/spec/schedule_types.h

```diff
@@ -244,6 +244,235 @@ typedef struct odp_schedule_capability_t {
+	odp_cache_stash_config_t config;
+
+} odp_cache_stash_prio_config_t;
+
+/**
+ * Schedule group parameters
+ */
+typedef struct odp_schedule_group_param_t {
+	/** Group specific cache stashing hints
+	 *
+	 *  Depending on the implementation, configuring these may improve
+	 *  performance. Cache stashing hints can be configured with a
+	 *  group-wide configuration using 'common' and with optional priority
+	 *  specific exceptions using 'prio' and 'num'. For example:
+	 *
+	 * @code{.unparsed}
+	 *    ...
+	 *    odp_schedule_group_param_t param;
+	 *    odp_cache_stash_prio_config_t prio;
+	 *
+	 *    odp_schedule_group_param_init(&param);
+	 *    prio.prio = 3;
+	 *    prio.config.regions.event_user_area_l2 = 1;
+	 *    prio.config.event_user_area.l2.offset = 0;
+	 *    prio.config.event_user_area.l2.len = 64;
+	 *    param.cache_stash_hints.prio = &prio;
+	 *    param.cache_stash_hints.num = 1;
+	 *    ...
+	 * @endcode
+	 *
+	 *  would disable cache stashing entirely for the group except
+	 *  priority 3 would have event user area L2 cache stashing
+	 *  enabled.
+	 *
+	 * @code{.unparsed}
+	 *    ...
+	 *    odp_schedule_group_param_t param;
+	 *    odp_cache_stash_prio_config_t prio;
+	 *
+	 *    odp_schedule_group_param_init(&param);
+	 *    param.cache_stash_hints.common.regions.event_data_l2 = 1;
+	 *    param.cache_stash_hints.common.event_data.l2.offset = 0;
+	 *    param.cache_stash_hints.common.event_data.l2.len = 128;
+	 *    prio.prio = 3;
+	 *    prio.config.regions.all = 0;
+	 *    param.cache_stash_hints.prio = &prio;
+	 *    param.cache_stash_hints.num = 1;
+	 *    ...
+	 * @endcode
+	 *
+	 *  would enable event data L2 cache stashing entirely for the
+	 *  group except disable cache stashing for priority 3.
+	 *
+	 * @code{.unparsed}
+	 *    ...
+	 *    odp_schedule_group_param_t param;
+	 *    odp_cache_stash_prio_config_t prio[2];
+	 *
+	 *    odp_schedule_group_param_init(&param);
+	 *    param.cache_stash_hints.common.regions.event_data_l2 = 1;
+	 *    param.cache_stash_hints.common.event_data.l2.offset = 0;
+	 *    param.cache_stash_hints.common.event_data.l2.len = 128;
+	 *    prio[0].prio = 3;
+	 *    prio[0].config.regions.event_data_l2 = 1;
+	 *    prio[0].config.event_data.l2.offset = 64;
+	 *    prio[0].config.event_data.l2.len = 128;
+	 *    prio[1].prio = 4;
+	 *    prio[1].config.regions.event_data_l2 = 1;
+	 *    prio[1].config.event_data.l2.offset = 64;
+	 *    prio[1].config.event_data.l2.len = 128;
+	 *    param.cache_stash_hints.prio = prio;
+	 *    param.cache_stash_hints.num = 2;
+	 *    ...
+	 * @endcode
+	 *
+	 *  would enable event data L2 cache stashing entirely for the
+	 *  group but priorities 3 and 4 would have event data cache
+	 *  stashing beginning from offset 64 instead of 0.
+	 */
+	struct {
+		/** Common group specific cache stashing hints
+		 *
+		 *  Configures cache stashing for each priority and queue under
+		 *  the group. By default, all regions are disabled (see
```

Comment:
Per queue config was removed for now. Spec would be more clear if "... and  queue ... " is removed.